### ISSUE TITLE
Include static assets for e-mail templates in Formbricks

### ISSUE THREAD

**Issue by salim-b** [formbricks/formbricks#4375]
### Is your feature request related to a problem? Please describe.

I noticed that static PNG images for Formbricks, Twitter and GitHub logos included in e-mails sent by Formbricks are currently fetched from an AWS S3 bucket:

https://github.com/formbricks/formbricks/blob/a3043c1f6d20324a4b681d353f35ed3a64ac68de/apps/web/modules/email/components/email-template.tsx#L17

This is subpar privacy-wise (at least when self-hosting Formbricks) since those assets could be used to track users. Also the images could in theory be replaced with rogue ones by the S3 bucket owner.

In other words: This S3 bucket is an unnecessary external dependency, especially when self-hosting Formbricks.

### Describe the solution you'd like

I suggest to include all necessary assets directly in Formbricks instead, so when an e-mail is rendered they're fetched from the same origin that is used by the Formbricks app, which is better privacy-wise (at least when self-hosting Formbricks).

It should be a minimal change. I'd be happy to submit a PR if you provide some guidance on where you'd like to place the PNG images in the repo.

**Comment by salim-b** [comment]
This has been fixed via https://github.com/formbricks/formbricks/pull/6782 👏

**Comment by jobenjada** [comment]
good lord, we've missed it for so long 🥺

you're a close observer of our repo, let's chat some time if you're up for it: cal.com/johannes/30
